README.md

```diff
@@ -305,6 +305,7 @@ _See also: [awesome-selfhosted/Database Management](https://github.com/awesome-s
 
 - [Diagrams.net](https://app.diagrams.net/) - A.K.A. [Draw.io](https://app.diagrams.net/). Easy to use Diagram UI with a plethora of templates.
 - [DrawThe.Net](http://go.drawthe.net/) - Javascript tool that uses a YAML-formatted input to programmatically create large, complex, and visually solid diagrams.
+- [Kroki](https://kroki.io) - API for generating diagrams from textual descriptions
```

Comment:
Missing several required fields (source code URL, license, language/platform, `.` after description), please refer to the Pull Request template and follow the recommended format.

Other entries in this section are also missing the required fields, because I did not have time to work on https://github.com/awesome-foss/awesome-sysadmin/issues/416 on this section yet.